docs/data/material/components/autocomplete/GloballyCustomizedOptions.tsx

```diff
@@ -0,0 +1,644 @@
+import * as React from 'react';
+import Autocomplete from '@mui/material/Autocomplete';
+import Box from '@mui/material/Box';
+import Stack from '@mui/material/Stack';
+import TextField from '@mui/material/TextField';
+import { createTheme, useTheme, ThemeProvider, Theme } from '@mui/material/styles';
+
+// Theme.ts
+const customTheme = (outerTheme: Theme) =>
+  createTheme({
+    palette: {
+      mode: outerTheme.palette.mode,
+    },
+    components: {
+      MuiAutocomplete: {
+        defaultProps: {
+          renderOption: (props, option, state, ownerState) => (
+            <Box sx={{ padding: '0 !important' }} component="li" {...props}>
+              <div
+                style={{
+                  fontSize: '12px',
+                  width: '100%',
+                  height: '100%',
+                  padding: '20px 24px',
+                  justifyContent: 'flex-start',
+                  textAlign: 'left',
+                }}
+              >
+                {ownerState.getOptionLabel?.(option)}
+              </div>
+            </Box>
```

Comment:
We can remove the unnecessary `div` and directly modify the Box (`<li>` element) instead:
```suggestion
            <Box
              sx={{
                borderRadius: '8px',
                margin: '5px',
                [`&.${autocompleteClasses.option}`]: {
                  padding: '8px',
                },
              }}
              component="li"
              {...props}
            >
              {ownerState.getOptionLabel?.(option)}
            </Box>
```